rustls-postquantum/examples/client.rs

```diff
@@ -0,0 +1,51 @@
+use std::io::{stdout, Read, Write};
```

Comment:
Can you add a short rustdoc comment describing the purpose of the example here?

Reply:
added a comment about the purpose and what to look for in its output